server/services/stats/side/teamcity_client.go

```diff
@@ -0,0 +1,38 @@
+package stats_side
+
+import (
+	"github.com/ivankorobkov/di"
+	"github.com/kapitanov/go-teamcity"
+)
+
+type (
+	TcClient interface {
+		GetProjects() ([]teamcity.Project, error)
+	}
+
+	tcClient struct {
+		client teamcity.Client
+	}
+)
+
+func TcClientModule(m *di.Module) {
+	m.AddConstructor(NewTcClient)
+	m.MarkPackageDep(Config{})
+}
+
+func NewTcClient(config Config) TcClient {
+	if config.TeamCityAddress == "test" {
+		return NewTestTcClient()
+	}
+
+	return &tcClient{
+		client: teamcity.NewClient(
+			config.TeamCityAddress,
+			teamcity.GuestAuth(),
```

Comment:
we need normal auth, not just guest

what if we will manage something automaticly few weeks later? 
if we use guest auth, we should rewrite module